Makefile

```diff
@@ -191,8 +191,9 @@ update-beats: update-beats-module update
 .PHONY: update-beats-module
 update-beats-module:
 	$(GO) get -d -u $(BEATS_MODULE)@$(BEATS_VERSION) && $(GO) mod tidy
-	diff -u .go-version $$($(GO) list -m -f {{.Dir}} $(BEATS_MODULE))/.go-version \
-		|| { code=$$?; echo ".go-version out of sync with Beats"; exit $$code; }
+	cp -f $$($(GO) list -m -f {{.Dir}} $(BEATS_MODULE))/.go-version .go-version
+	sed -i'.bck' -E -e "s#(FROM golang):[0-9]+\.[0-9]+\.[0-9]+#\1:$$(cat .go-version)#g" tests/Dockerfile
+	sed -i'.bck' -E -e "s#(FROM golang):[0-9]+\.[0-9]+\.[0-9]+#\1:$$(cat .go-version)#g" Dockerfile
```

Comment:
can/should we also update docs/version.asciidoc here?

https://github.com/elastic/apm-server/blame/master/docs/version.asciidoc#L3